src/elements/common/selectors/__tests__/version.js

```diff
@@ -7,12 +7,19 @@ import {
     VERSION_UPLOAD_ACTION,
 } from '../../../../constants';
 
+import { FILE_REQUEST } from '../../../../features/file-request/constants';
+import messages from '../../../../features/file-request/messages';
```

Comment:
We should avoid importing feature-specific modules into a common module, if possible.